packages/nextjs/src/shared/url.ts

```diff
@@ -0,0 +1,25 @@
+// TODO: Use the same function defined in @clerk/shared once the package is tree shakeable
```

Comment:
❓ What's the latest update on this? Is @clerk/shared still not tree-shakeable?

Reply:
I will investigate it as part of https://github.com/clerkinc/javascript/pull/1767

Reply:
It seems like `@clerk/shared` is not entirely tree-shakeable due to `window.__SWR_DEVTOOLS_USE__ = undefined;`
I used the `@next/bundle-analyzer` and ran our A`NALYZE=true npm run build` to generate the bundle analysis below (the screenshot is part of the client.html analysis).
<img width="491" alt="Screenshot 2023-09-27 at 17 49 19" src="https://github.com/clerkinc/javascript/assets/3936408/533ccc7b-00ca-4483-ad9f-be27c7c65e0e">

```
// next.config.js
const withBundleAnalyzer = require('@next/bundle-analyzer')({
  enabled: process.env.ANALYZE === 'true',
})
module.exports = withBundleAnalyzer(nextConfig) 
```

I've created a discussion in the team channel to resolve this.

ref: https://github.com/clerkinc/javascript/blob/0a17322842393670344369b0217ed3e989f912bb/packages/shared/src/hooks/clerk-swr.ts#L4C1-L4C1

Reply:
It seems that tree-shaking is fixed. 
The above dependency still exists in the client analysis because it is being used in some `@clerk/clerk-react` hooks that are being used by the clerk UI components.